README.md

```diff
@@ -86,26 +88,34 @@ Table of Contents
 
 
 ### • Automation
-- [**Easer**](https://github.com/renyuneyun/Easer) <sup>**[[F-Droid](https://f-droid.org/app/ryey.easer)]**</sup>
 - [**Automation**](https://git.server47.de/jens/Automation) <sup>**[[F-Droid](https://f-droid.org/app/com.jens.automation2)]**</sup>
+- [**Easer**](https://github.com/renyuneyun/Easer) <sup>**[[F-Droid](https://f-droid.org/app/ryey.easer)]**</sup>
 
 
 ### • Browser
-- [**Firefox**](https://www.mozilla.org/firefox/android/)
-- [**Fennec**](https://hg.mozilla.org/releases/mozilla-esr68/) <sup>**[[F-Droid](https://f-droid.org/app/org.mozilla.fennec_fdroid)]**</sup>
-- [**FFUpdater**](https://notabug.org/Tobiwan/ffupdater) <sup>**[[F-Droid](https://f-droid.org/app/de.marmaro.krt.ffupdater)]**</sup>
+#### = Downloaders =
 - [**Chromium SWE Updater**](https://forum.xda-developers.com/android/apps-games/app-code-aurora-s-chromium-swe-browser-t3603932) <sup>**[[F-Droid](https://f-droid.org/app/chromiumupdater.bamless.com.chromiumsweupdater)]**</sup>
-- [**Lightning**](http://acrdevelopment.org/) <sup>**[[F-Droid](https://f-droid.org/app/acr.browser.lightning)]**</sup>
+- [**FFUpdater**](https://notabug.org/Tobiwan/ffupdater) <sup>**[[F-Droid](https://f-droid.org/app/de.marmaro.krt.ffupdater)]**</sup>
+
+#### = Based on Firefox =
+- [**Fennec**](https://hg.mozilla.org/releases/mozilla-esr68/) <sup>**[[F-Droid](https://f-droid.org/app/org.mozilla.fennec_fdroid)]**</sup>
+- [**Firefox**](https://www.mozilla.org/firefox/android/)
+- [**Iceraven**](https://github.com/fork-maintainers/iceraven-browser) <sup>**[[FFUpdater](https://notabug.org/Tobiwan/ffupdater)]**</sup>
+- [**Mull**](https://gitlab.com/divested-mobile/mull-fenix) <sup>**[[F-Droid](https://f-droid.org/app/us.spotco.fennec_dos)]**</sup>
+
+#### = Based on Chromium or WebView =
 - [**Bromite**](https://www.bromite.org/)
 - [**DuckDuckGo Privacy Browser**](https://github.com/duckduckgo/Android) <sup>**[[F-Droid](https://f-droid.org/app/com.duckduckgo.mobile.android)]**</sup>
-- [**SmartCookieWeb**](https://github.com/CookieJarApps/SmartCookieWeb) <sup>**[[F-Droid](https://f-droid.org/app/com.cookiegames.smartcookie)]**</sup>
-- [**Privacy Browser**](https://gitweb.stoutner.com/?p=PrivacyBrowser.git) <sup>**[[F-Droid](https://f-droid.org/app/com.stoutner.privacybrowser.standard)]**</sup>
+- [**Kiwi Browser**](https://kiwibrowser.com/)
```

Comment:
> I'm not familiar with Kiwi; is it really an Open Source project?

https://github.com/Tobi823/ffupdater/issues/35
It is supposedly spywear. I am not a dev, so can someone confirm this issue is legit for me?

Reply:
Thanks for the link. From the discussion it looks very suspicious for me. Better don't add it (at least for now).

Reply:
I'm not a developer either.
https://github.com/kiwibrowser/src/blob/master/LICENSE

Kiwi source code is open but as they haven't mentioned  any [GPL-Compatible](https://www.gnu.org/licenses/license-list.html) or Apache license. No lo FOSS license means project not opensource. License is important, F-Droid always check license 1st ( check their [Inclusion-policy](https://f-droid.org/en/docs/Inclusion_Policy/))

Plus check: https://github.com/kiwibrowser/src#business-model

Reply:
Thanks, the license looks like a BSD 4-clause. GH probably doesn't recognise it because of the `//`.

However, due to the linked business model it's … debatable …

If there's no strong opinion for adding it, I'd suggest leaving it out for now.

Btw. there's RFP on F-Droid: https://gitlab.com/fdroid/rfp/-/issues/906